src/components/note/Note.jsx

```diff
@@ -1,39 +1,52 @@
 import React from "react";
 import {
   ArchiveIcon,
-  ColorPalleteIcon,
-  LabelIcon,
+  EditIcon,
   PinLineIcon,
   TrashIcon,
 } from "../../assests/icons/icons";
+import { useAuthContext, useNoteContext } from "../../context";
 import "./Note.css";
-export function Note() {
+export function Note({ notes, setEditNoteBtn }) {
+  const { deleteNoteHandler, setNoteDetail, setEditNoteDetail } = useNoteContext();
+  const { token } = useAuthContext();
   return (
-    <div className="note-container">
-      <h2 className="note-title">
-        Title of note
-        <span className="note-pin ft-grey">
-          <PinLineIcon />
-        </span>
-      </h2>
-      <p className="note-content">content of note</p>
-      <div className="note-footer">
-        <div className="note-date">Created on 02/05/2022</div>
-        <div className="note-icons ft-grey">
-          <span>
-            <ColorPalleteIcon />
-          </span>
-          <span>
-            <LabelIcon />
-          </span>
-          <span>
-            <ArchiveIcon />
-          </span>
-          <span>
-            <TrashIcon />
-          </span>
-        </div>
-      </div>
-    </div>
+    <>
+      {notes.map((note) => (
+        <li key={note._id} className="m-b-1">
+          <div
+            className="note-container"
+            style={{ backgroundColor: note.cardColor }}
+          >
+            <h3 className="note-title">
+              {note.title}
+
+              <span className="note-pin ft-grey">
+                <PinLineIcon />
+              </span>
+            </h3>
+            <p className="note-content">{note.desc}</p>
+            <div className="note-footer">
+              <div className="note-date">
+                Created on {new Date().toLocaleDateString()}
+                <span className="note-chip">{note.priority}</span>
+                <span className="note-chip">{note.tags}</span>
+              </div>
+              <div className="note-icons ft-grey">
+                <span onClick={()=> { setEditNoteDetail(note); setNoteDetail(note); setEditNoteBtn(true) } }>
```

Comment:
As, you are using onClick here, so see if you can use button tag here. As it will make the code for semantic and also will increase the app accessibility.

Reply:
yes surely changed to btn